packages/sdks/mitosis.config.cjs

```diff
@@ -763,6 +763,27 @@ const ANGULAR_MARK_SAFE_INNER_HTML = () => ({
   },
 });
 
+/**
+ * Angular doesn't support hydration for components created dynamically.
+ * Refer: https://angular.dev/errors/NG0503
+ * GitHub issue: https://github.com/angular/angular/issues/51798
```

Comment:
Does this issue resolve content duplication as well like https://builder-io.atlassian.net/browse/ENG-8042 ?